plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -155,11 +204,20 @@ protected void performDefaults()
 
         notificationsEnabled.setSelection(store.getBoolean(ModelPreferences.NOTIFICATIONS_ENABLED));
         notificationsCloseDelay.setSelection(store.getInt(ModelPreferences.NOTIFICATIONS_CLOSE_DELAY_TIMEOUT));
-
+        if (store.getString(DBeaverPreferences.CLIENT_TIMEZONE).equals("DEFAULT")) {
+            clientTimezone.setText(store.getString(DBeaverPreferences.CLIENT_TIMEZONE));
+        } else {
+            clientTimezone.setText(TimezoneRegistry.addGMTTime(store.getString(DBeaverPreferences.CLIENT_TIMEZONE)));
+        }
```

Comment:
Three store lookups. There's a feature in Java called _local variables_, you should google it.